modules/claims_api/spec/sidekiq/evidence_waiver_builder_job_spec.rb

```diff
@@ -21,6 +21,14 @@
     end
   end
 
+  describe 'when an errored job has a time limitation' do
+    it 'logs to the ClaimsApi Logger' do
```

Comment:
Here and below: are we actually asserting the logs were written? It seems we are only checking that the job is expired.

Reply:
Ya 100%, adjusting that text